### ISSUE TITLE
Cannot run list if missing Python versions

### ISSUE THREAD

**Issue by jd** [DataDog/riot#106]
With only Python 3.9 available running `riot list` end up with:
```
Traceback (most recent call last):
  File "/usr/local/bin/riot", line 5, in <module>
    from riot.__main__ import main
  File "/usr/local/lib/python3.9/site-packages/riot/__main__.py", line 3, in <module>
    main(auto_envvar_prefix="RIOT")
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/riot/cli.py", line 76, in list_venvs
    ctx.obj["session"].list_venvs(
  File "/usr/local/lib/python3.9/site-packages/riot/riot.py", line 413, in list_venvs
    if not venv_pattern.search(inst.venv_path()):
  File "/usr/local/lib/python3.9/site-packages/riot/riot.py", line 233, in venv_path
    base_path = self.py.venv_path()
  File "/usr/local/lib/python3.9/site-packages/riot/riot.py", line 102, in venv_path
    version = self.version().replace(".", "")
  File "/usr/local/lib/python3.9/site-packages/riot/riot.py", line 76, in version
    path = self.path()
  File "/usr/local/lib/python3.9/site-packages/riot/riot.py", line 98, in path
    raise FileNotFoundError(f"Python interpreter {self._hint} not found")
FileNotFoundError: Python interpreter 3.8 not found
```
I'm just asking for the list so it should not complain.

**Comment by brettlangdon** [comment]
Would be interesting to report when listing which interpreters are available vs not rather than bailing.

`Interpreter(_hint=3.8, available=True)`

**Comment by Kyle-Verhoog** [comment]
A solution to this was merged in #116
